UI/MapboxNavigationUI/RouteStepFormatter.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import Mapbox
+import MapboxDirections
+
+class RouteStepFormatter: Formatter {
+    let instructions = OSRMInstructionFormatter(version: "v5")
+    
+    override func string(for obj: Any?) -> String? {
+        return string(for: obj, markUpWithSSML: false)
+    }
+    
+    func string(for obj: Any?, markUpWithSSML: Bool) -> String? {
```

Comment:
This SSML markup code is unused. I take it it’s for the Polly integration being added in #42.